app/api/helpers/custom_fields.py

```diff
@@ -27,8 +27,8 @@ def format(self, value):
         works only for GET requests
         """
         if not self.validate(value):
-            print 'Validation of field with value \"%s\" (%s) failed' % (
-                value, str(self.__class__.__name__))
+            #print 'Validation of field with value \"%s\" (%s) failed' % (
```

Comment:
Travis is failing because of an empty if block. Remove it altogether. We can always add back when needed.